lib/elixir/lib/exception.ex

```diff
@@ -240,7 +240,7 @@ defmodule Exception do
   defp format_exit_reason(:noconnection), do: "no connection"
 
   defp format_exit_reason(:noproc) do
-    "no process"
+    "no process: the process is dead or no process is associated with the given name, possibly because its application isn't started"
```

Comment:
Good points. I changed it to this – please let me know what you think:

"no process: there is no process that is currently alive and associated with the given name, possibly because its application isn't started"